lib/apiaria.rb

```diff
@@ -28,4 +28,27 @@ def get_mentor_availability(mentor_id)
     JSON.parse(response.body)
   end
   
+  def get_messages(pageNum = nil)
+    unless pageNum
+      response = self.class.get("/message_threads",  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    else
+      url = "/message_threads?page=" + pageNum.to_s
+      response = self.class.get(url,  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    end
+  end
+
+  def create_message(sender, recipientId, body, options = {})
```

Comment:
Hello @praveenyr I'm getting the messages consistently. Let's go with the code you have now and we can adjust once you get to the testing phase of the project (extra credit which will be required of you). I'll be boarding a plane soon so if you PR and I'm not available, feel free to move forward.